hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```diff
@@ -77,8 +78,13 @@ public static ContainerCommandResponseProto logAndReturnError(
       ContainerCommandRequestProto request) {
     String logInfo = "Operation: {} , Trace ID: {} , Message: {} , " +
         "Result: {} , StorageContainerException Occurred.";
-    log.info(logInfo, request.getCmdType(), request.getTraceID(),
-        ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
+    if (ex.getResult() == CLOSED_CONTAINER_IO) {
+      log.debug(logInfo, request.getCmdType(), request.getTraceID(),
+          ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
```

Comment:
Nit: Adding `isDebugEnabled()` check would let us avoid some unnecessary operations.

```suggestion
      if (log.isDebugEnabled()) {
        log.debug(logInfo, request.getCmdType(), request.getTraceID(),
            ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
      }
```